Wabbajack.VirtualFileSystem/Context.cs

```diff
@@ -336,7 +328,7 @@ public class KnownFile
 
     public class DisposableList<T> : List<T>, IDisposable
```

Comment:
Random side tangent, but might be nice to start moving these classes which are pretty reusable and might be helpful for other code.. into a `Utility` folder?   Maybe `Wabbajack.Common/Util`?